Lesson22/Anil/todo-app/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>To-Do App</title>
+		<link rel="stylesheet" type="text/css" href="styles.css" />
+	</head>
+	<body>
+
+		<button id="btnSayHello">Say hello</button>
```

Comment:
No need for this and the next buttons, they were just examples in the lesson. Let's remove them.

Reply:
ı removed the buttons